express/src/controllers/manage-account.ts

```diff
@@ -5,12 +5,30 @@ import {User} from "../../@types/User";
 import {Service} from "../../@types/Service";
 
 export const listServices = async function(req: Request, res: Response) {
-    res.render('manage-account/list-services.njk');
+    const lambdaFacade: LambdaFacadeInterface = req.app.get("lambdaFacade");
+    if(req.session.selfServiceUser == undefined) {
+        res.render('there-is-a-problem.njk');
+        return;
+    }
+    const user: User = req.session.selfServiceUser;
+    const services = await lambdaFacade.listServices(user.pk.S as string, req.session.authenticationResult?.AccessToken as string);
```

Comment:
Is it possible for us to abstract the data that frontend gets so it does not have to get the value from `pk.S`.

Reply:
If not now then please include for later.

Reply:
Yeah - that was my quick and dirty, make it work solution.

We can use the aws utility unmarshall (or marshall I always forget which is which) - the question is whether we do it in the lambda that returns the DB result or in the facade that calls the lambda from within Express.  We should discuss it to choose which one we want to do.